rest_framework/exceptions.py

```diff
@@ -101,10 +101,8 @@ class APIException(Exception):
     default_code = 'error'
 
     def __init__(self, detail=None, code=None):
-        if detail is None:
-            detail = self.default_detail
-        if code is None:
-            code = self.default_code
+        detail = detail or self.default_detail
+        code = code or self.default_code
```

Comment:
I would revert these changes. `or` will evaluate falsey conditions (`None`, `''`, `[]`, etc...) which is different from the existing `is None` check.

Reply:
Oh.. you're right. thanks!